### ISSUE TITLE
Icon request: fa-line-wrap

### ISSUE THREAD

**Issue by valtlai** [FortAwesome/Font-Awesome#2290]
Hey!

I need word wrap icon. I would use it in “enable/disable line wrap” button on code boxes of my site.

It could look like this:
![line wrap icon](https://cloud.githubusercontent.com/assets/5496284/3841241/1b30f8b6-1e29-11e4-9cbc-f9e61a809616.png)

**Comment by dimitrieh** [comment]
I was looking forward to getting this implemented as well. However I created a different implementation:
![image](https://cloud.githubusercontent.com/assets/429659/16691971/34403cca-452f-11e6-9efe-dd5154e0060e.png)



Download: [linewrap.pdf](https://github.com/FortAwesome/Font-Awesome/files/354543/linewrap.pdf)

**Comment by mouse0270** [comment]
Well, since this hasn't seen much of an update, thought I would place it here:


![](https://cdn4.iconfinder.com/data/icons/ui-controls-editor/32/interface-controls-text-wrap-16.png)
![](https://cdn4.iconfinder.com/data/icons/ui-controls-editor/32/interface-controls-text-wrap-32.png)
![](https://cdn4.iconfinder.com/data/icons/ui-controls-editor/32/interface-controls-text-wrap-64.png)
![](https://cdn4.iconfinder.com/data/icons/ui-controls-editor/32/interface-controls-text-wrap-128.png)

**Comment by trdyer** [comment]
I would love a linewrap/softwrap icon!

**Comment by omenking** [comment]
Would love such an icon

**Comment by kevinirlen** [comment]
this has been abandonned?

**Comment by tagliala** [comment]
> this has been abandonned?

Nope, please take a look at our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work

Closing here

**Comment by somethingSTRANGE** [comment]
I was just looking for this icon earlier today. I'd love to see this added.

**Comment by matsn0w** [comment]
+1

**Comment by alexec** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by 6TELOIV** [comment]
Would love this one; super helpful as a toggle button for wrapping code, as well as any preformatted text on mobile screens.

**Comment by 6TELOIV** [comment]
Lucide's icon, for inspiration :) https://lucide.dev/icons/wrap-text
